monospacifier.py

```diff
@@ -62,6 +62,8 @@ def needs_scaling(glyph):
 
     @staticmethod
     def set_width(glyph, width):
+        uni = glyph.unicode
+        width *= (1 + math.floor(math.log(uni, 256))) if 0 <= uni <= sys.maxunicode else (1)
```

Comment:
Not all multi-byte characters are two characters wide, are they?

Reply:
I can't exactly tell because I'm not an Unicode professional.

However according to https://www.unicode.org/reports/tr11/#Overview -

> When dealing with East Asian text, there is the concept of an inherent width of a character. This width takes on either of two values: narrow or wide. For traditional mixed-width East Asian legacy character sets, this classification into narrow and wide corresponds with few exceptions directly to the storage size for each character: a few narrow characters use a single byte per character and all other characters (usually wide) use two or more bytes.

Reply:
Isn't that sentence about legacy charsets?  To be concrete, all the non-alphabetic symbols shown in the gif in the README take two or three bytes, so unless I missing something codepoints won't be enough.

According to the doc you found, though, we need to consult the East_Asian_Width property:

> The Unicode character property East_Asian_Width provides a default classification of characters, which an implementation can use to decide at runtime whether to treat a character as narrow or wide.